yarn.lock

```diff
@@ -1022,14 +1022,12 @@
   resolved "https://registry.yarnpkg.com/@csstools/convert-colors/-/convert-colors-1.4.0.tgz#ad495dc41b12e75d588c6db8b9834f08fa131eb7"
   integrity sha512-5a6wqoJV/xEdbRNKVo6I4hO3VjyDq//8q2f9I6PBAvMesJHFauXDorcNCsr9RzvsZnaWi5NYCcfyqP1QeFHFbw==
 
-"@dhis2/analytics@2.4.7":
-  version "2.4.7"
-  resolved "https://registry.yarnpkg.com/@dhis2/analytics/-/analytics-2.4.7.tgz#0cbe4e44db69e690b08a2b284ce08b630142d73f"
-  integrity sha512-mESuLf4x224WacDRGju6i5GIVQgtEghHR/tCxtqref1kT1o4NN+ZGjbBn7+fmE7FqIjBt7J0/1/IR58Fq0+GRg==
+"@dhis2/analytics@2.4.12":
```

Comment:
We're now requesting both `@dhis2/analytics@2.4.12` (from Dashboards) and `@dhis2/analytics@2.4.9` (from the DV plugin) explicitly. Can we start using patch `~` in both places so we get `@dhis2/analytics@~2.4.12` etc so they can resolve to the same version? To avoid needing two versions of Analytics in Dashboards in the future (once DV plugin also requests a `~` patch. @janhenrikoverland @amcgee

Reply:
To avoid multiple Analytics versions, I suggest https://github.com/dhis2/data-visualizer-app/pull/967 is merged first along with a new version of the plugin (which uses `@dhis2/analytics@~2.4.12`). This new plugin version can then be used here along with `@dhis2/analytics@~2.4.12` to resolve both the plugin Analytics and Dashboards Analytics to `2.4.12`